### DISCUSSION TITLE
Install 2.0.0 rc3 on react-native project

### DISCUSSION THREAD

**Discussion by nmateo** [vercel/swr#2269] [General] [answered]
I'm stuck on installing the release candidate [2.0.0-rc.3](https://github.com/vercel/swr/releases/tag/2.0.0-rc.3)
I tried multiple ways but at best i'm stuck when running the project:
![image](https://user-images.githubusercontent.com/9270970/206250647-069d353b-fc54-4658-9410-7078e4f878c4.png)

Can someone provides step to install release candidates please, thanks in advance ?

**Comment by huozhi** [comment, marked as answer]
I can see the `core/dist/index.js` is successfully published https://unpkg.com/swr@2.0.0-rc.3/core/dist/index.js

Do you have any minimal reproduction for it?

**Reply by nmateo** [reply]
> I can see the `core/dist/index.js` is successfully published https://unpkg.com/swr@2.0.0-rc.3/core/dist/index.js
> 
> Do you have any minimal reproduction for it?

Nevermind i managed to install it, i tried to install it from source because i did not see i could install it with npm i swr@2.0.0-rc.3,  thanks anyway!
